File: Python/Estrutura sequencial/circulo.py
# Importando biblioteca matemática
import math

# Variáveis
# Em Python não é necessário fazer esse passo...
r: float
area: float

# Entrada de dados
r = float(input("Digite o valor do raio do círculo: "))

# Artíficio matemático
area = math.pi * r ** 2

# Saída de dados
print(f"A área do círculo de raio {r} é {area:.3f}")

File: C++/Estrutura sequencial/circulo.cpp
// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    double r, area;

    // Entrada de dados
    cout << "Digite o valor do raio do circulo: ";
    cin >> r;

    // Algoritmo matemático
    area = (double) 3.14159 * pow(r, 2);

    // Saída de dados
    cout << fixed << setprecision(3); // Quantidade de casas decimais
    cout << "A area do circulo e " << area;

    return 0;
}